src/main/resources/assets/react/documentTypes/NewOrEditDocumentType.jsx

```diff
@@ -172,6 +228,7 @@ export function NewOrEditDocumentType({
 					addFields,
 					properties
 				};
+
 				if (_id) {
 					variables._id = _id;
 					variables._versionKey = initialValues._versionKey;
```

Comment:
unsure if this is actually used now? If the fetch is send twice ( like a failure) it can crash?